### ISSUE TITLE
Sorry, nobody on Reddit goes by that name

### ISSUE THREAD

**Issue by ghost** [cubicbyte/reddit-account-generator#23]
When I registered an account, after a few days the account was locked with the message:

Sorry, nobody on Reddit goes by that name
This account may have been banned or the username is incorrect

Even if the reddit account is still alive, when posting, the reddit account will be locked.

Is there any way to solve this problem?

Meanwhile, I bought other Reddit accounts online and some of their accounts were still able to post.

I use rotating IP.

**Comment by cubicbyte** [comment]
It looks like this may be happening for the following reasons:

- IP detection
- Email detection - too many accounts per account or suspicious mail (if using random mails)
- Suspicious activity - like an account just created and immediately makes posts.

But unfortunately I haven't been able to reproduce the account ban yet. But I'd appreciate the help with that! You can PM me at email/telegram in my profile

**Comment by IqBroly** [comment]
> It looks like this may be happening for the following reasons:
> 
>     * IP detection
> 
>     * Email detection - too many accounts per account or suspicious mail (if using random mails)
> 
>     * Suspicious activity - like an account just created and immediately makes posts.
> 
> 
> But unfortunately I haven't been able to reproduce the account ban yet. But I'd appreciate the help with that! You can PM me at email/telegram in my profile

I used ip addresses from mobile proxy, all different but 50% got banned immediately, the rest got banned after 1-2 days. didn't even login to the accounts. If some account is not banned somehow it's immediately banned after commenting

**Comment by hmm-star** [comment]
hi
so does that mean that this script does not work ?

**Comment by cubicbyte** [comment]
> hi so does that mean that this script does not work ?

No, the script works, but if you have a bad proxy, accounts can be quickly banned
